roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+# defaults file for openshift_prometheus
+openshift_prometheus_state: present
+
+openshift_prometheus_namespace: prometheus
+
+openshift_prometheus_replicas: 1
+openshift_prometheus_node_selector: {"region":"infra"}
+
+# container resources
+openshift_prometheus_cpu_limit_prometheus: null
```

Comment:
Consider inverting the naming convention to have the 'component' before the 'property' which is what we did with logging.  It allows for a more natural grouping of settings by subcomponent: 
`openshift_prometheus_alert_buffer_cpu_limit`